utils/dump_taxon_tree.pl

```diff
@@ -265,33 +269,34 @@ sub node_to_dynatree {
   
   if (@{$node->dba}) {
     foreach my $dba (@{$node->dba}) {
-      my $display_name = get_node_display_name($dba, $name);
+      my $meta = get_meta($dba, $name);
       push @output, {  
-        key   => ucfirst($dba->species),
-        title => $display_name
+        key   => $meta->{'prod_name'},
+        title => $meta->{'display_name'}
       };
     }
   }  
   
   return @output;
 }
 
-sub get_node_display_name {
+sub get_meta {
   # this subroutine should not be necessary; we expect each node to have core adaptors, including the MetaContainer,
   # but at the same time, we are a bit nervous to trust the Perl API entirely
   my ($node_dba, $fallback_name) = @_;
   my $meta_adaptor = Bio::EnsEMBL::Registry->get_adaptor( $node_dba->species, "core", "MetaContainer" );
+  my $meta = {};
+
   if ($meta_adaptor) {
-    my $display_name = $meta_adaptor->get_display_name();
+    $meta->{'display_name'} = $meta_adaptor->get_display_name();
+    $meta->{'prod_name'} = $meta_adaptor->single_value_by_key('species.production_name');
+
     $meta_adaptor->dbc->disconnect_if_idle();
```

Comment:
This line feels like it should be just before the return.

Reply:
No, If it didnt return a meta_adaptor above then the call to dbc become undefined and will throw an error.

Reply:
But thats just the database connection. Its basically you are killing the db connection but the dangerous about this, if you dont kill it at the end of the function, it can stay open since its only within this if statement

Reply:
sorry i understand your point.  ignore my previous comment